src/events/Monitors.cpp

```diff
@@ -114,6 +114,24 @@ void Events::listener_newOutput(wl_listener* listener, void* data) {
             }
         }
     }
+
+    static auto PCURSORMONITOR    = CConfigValue<std::string>("general:default_cursor_monitor");
+    static auto firstMonitorAdded = std::chrono::system_clock::now();
+    static bool cursorDefaultDone = false;
+
+    if (cursorDefaultDone || *PCURSORMONITOR == STRVAL_EMPTY)
+        return;
+
+    auto timePassedSec = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now() - firstMonitorAdded);
+    if (timePassedSec.count() > 10)
+        cursorDefaultDone = true;
```

Comment:
should also return, otherwise setting this at a random point will warp the cursor once.